tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -129,8 +125,8 @@ def test_get_metrics_and_expectations(
         ExpectationConfiguration(
             **{
                 "kwargs": {
-                    "min_value": 10000.0,
-                    "max_value": 10000.0,
+                    "min_value": 10000,
```

Comment:
This was the reason why these variables exist. Each Rule governs its own variables. now the value can be an integer, rather than 10000.0 (which is what we had here before) :)